mapviz/src/mapviz.cpp

```diff
@@ -201,7 +209,23 @@ void Mapviz::Initialize()
       connect(&spin_timer_, SIGNAL(timeout()), this, SLOT(SpinOnce()));
     }
 
-    node_ = new ros::NodeHandle("mapviz");
+    node_ = new ros::NodeHandle("~");
```

Comment:
Explicitly setting this to `mapviz` would cause the node handle to use "mapviz" as its name even if the actual mapviz node had a different name, which it normally would since Mapviz's node name is anonymized.  This would make it rather difficult to set/get params directly on this node.  Setting it to `~` instead will make it use the node's real name.